besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1320,6 +1323,13 @@ static class TxPoolOptionGroup {
           "Specifies the maximum number of blocks to retrieve logs from via RPC. Must be >=0. 0 specifies no limit  (default: ${DEFAULT-VALUE})")
   private final Long rpcMaxLogsRange = 5000L;
 
+  @CommandLine.Option(
+      hidden = true,
+      names = {"--p2p-peer-lower-bound"}, // TODO: do we want to make this not experimental
```

Comment:
if so - needs a changelog - this is a breaking change - and docs

Reply:
DONE